rcongui/.env

```diff
@@ -1 +1 @@
-REACT_APP_API_URL=http://127.0.0.1:8000/api/
+REACT_APP_API_URL=http://localhost:3000/api/
```

Comment:
Do we want to change this?

Reply:
Yeah, the backend runs on 8000, however, there is a proxy in vite.config.js, which proxies the api requests to the backend running on 8000 :)
I usually do it like that, as it (from my point of view) is more natural to look for the API url in the config there. And I use that proxy mechanism, when I want to hook up my local frontend with a live backend (without running that locally). Doing it with the vite proxy allows me to go around CORS 🤣